corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -86,13 +87,17 @@ def update_sync_log_with_checks(sync_log, xform, cases, case_db,
     try:
         sync_log.update_phone_lists(xform, cases)
     except SyncLogAssertionError, e:
+        _assert = soft_assert(to=['czue' + '@' + 'dimagi.com'])
+        _assert(False, 'Sync log assertion error {} for log {}, user {}, form {}, domain {}'.format(
+            e, sync_log._id, sync_log.user_id, xform._id, xform.domain
+        ))
```

Comment:
I suggest doing

```python
_assert(False, 'Sync log assertion error', 
       {'log': sync_log._id, 'user': sync_log.user_id, 'form': xform._id, 'domain': xform.domain, 'error': e})
```
That'll make the emails thread together by having the same subject, and then the extra data you're interested in will show up in the individual email bodies.